embassy-stm32/src/i2c/mod.rs

```diff
@@ -13,6 +16,7 @@ use embassy_hal_internal::{into_ref, Peripheral, PeripheralRef};
 use embassy_sync::waitqueue::AtomicWaker;
 #[cfg(feature = "time")]
 use embassy_time::{Duration, Instant};
+use stm32_metapac::i2c::vals::Oamsk;
```

Comment:
Odd error in CI, needs a deeper look.

Reply:
I haven't been able to figure out why I am getting this error in CI.

Reply:
May be @Dirbaio can help, he's very familiar with his CI setup...